.travis.yml

```diff
@@ -25,49 +23,51 @@ os:
   # unoptimized routines. See https://www.openssl.org/docs/man1.1.0/crypto/OPENSSL_ia32cap.html
-  - OPENSSL_ia32cap="~0x200000200000000" BUILD_S2N=true TESTS=integration
-  - LATEST_CLANG=true TESTS=fuzz
-  - TESTS=sawHMAC SAW_HMAC_TEST=md5 SAW=true
-  - TESTS=sawHMAC SAW_HMAC_TEST=none SAW=true
-  - TESTS=sawHMAC SAW_HMAC_TEST=sha1 SAW=true
-  - TESTS=sawHMAC SAW_HMAC_TEST=sha224 SAW=true
-  - TESTS=sawHMAC SAW_HMAC_TEST=sha256 SAW=true
-  - TESTS=sawHMAC SAW_HMAC_TEST=sha384 SAW=true
-  - TESTS=sawHMAC SAW_HMAC_TEST=sha512 SAW=true
-  - TESTS=sawDRBG SAW=true
-  - TESTS=sawHMACFailure SAW=true
+  - S2N_LIBCRYPTO=openssl-1.1.0 OPENSSL_ia32cap="~0x200000200000000" BUILD_S2N=true TESTS=integration GCC6_REQUIRED=true
+  - S2N_LIBCRYPTO=openssl-1.1.0 LATEST_CLANG=true TESTS=fuzz
+  - TESTS=sawHMAC SAW_HMAC_TEST=md5    SAW=true GCC6_REQUIRED=true
+  - TESTS=sawHMAC SAW_HMAC_TEST=none   SAW=true GCC6_REQUIRED=true
+  - TESTS=sawHMAC SAW_HMAC_TEST=sha1   SAW=true GCC6_REQUIRED=true
+  - TESTS=sawHMAC SAW_HMAC_TEST=sha224 SAW=true GCC6_REQUIRED=true
+  - TESTS=sawHMAC SAW_HMAC_TEST=sha256 SAW=true GCC6_REQUIRED=true
+  - TESTS=sawHMAC SAW_HMAC_TEST=sha384 SAW=true GCC6_REQUIRED=true
+  - TESTS=sawHMAC SAW_HMAC_TEST=sha512 SAW=true GCC6_REQUIRED=true
+  - TESTS=sawDRBG                      SAW=true
+  - TESTS=sawHMACFailure               SAW=true
 
 matrix:
   exclude:
   - os: osx
-    env: TESTS=sawHMAC SAW_HMAC_TEST=md5 SAW=true
+    env: TESTS=sawHMAC SAW_HMAC_TEST=md5    SAW=true GCC6_REQUIRED=true
   - os: osx
-    env: TESTS=sawHMAC SAW_HMAC_TEST=none SAW=true
+    env: TESTS=sawHMAC SAW_HMAC_TEST=none   SAW=true GCC6_REQUIRED=true
   - os: osx
-    env: TESTS=sawHMAC SAW_HMAC_TEST=sha1 SAW=true
+    env: TESTS=sawHMAC SAW_HMAC_TEST=sha1   SAW=true GCC6_REQUIRED=true
   - os: osx
-    env: TESTS=sawHMAC SAW_HMAC_TEST=sha224 SAW=true
+    env: TESTS=sawHMAC SAW_HMAC_TEST=sha224 SAW=true GCC6_REQUIRED=true
   - os: osx
-    env: TESTS=sawHMAC SAW_HMAC_TEST=sha256 SAW=true
+    env: TESTS=sawHMAC SAW_HMAC_TEST=sha256 SAW=true GCC6_REQUIRED=true
   - os: osx
-    env: TESTS=sawHMAC SAW_HMAC_TEST=sha384 SAW=true
+    env: TESTS=sawHMAC SAW_HMAC_TEST=sha384 SAW=true GCC6_REQUIRED=true
   - os: osx
-    env: TESTS=sawHMAC SAW_HMAC_TEST=sha512 SAW=true
+    env: TESTS=sawHMAC SAW_HMAC_TEST=sha512 SAW=true GCC6_REQUIRED=true
   - os: osx
-    env: LATEST_CLANG=true TESTS=fuzz
+    env: S2N_LIBCRYPTO=openssl-1.1.0 LATEST_CLANG=true TESTS=fuzz
   - os: osx
-    env: TESTS=sawDRBG SAW=true
+    env: TESTS=sawDRBG                      SAW=true  GCC6_REQUIRED=true
   - os: osx
-    env: TESTS=sawHMACFailure SAW=true
+    env: TESTS=sawHMACFailure               SAW=true
   #This exception is because the test isn't finished yet, remove to turn on DRBG Saw tests
-  - env: TESTS=sawDRBG SAW=true
+  - env: TESTS=sawDRBG                      SAW=true
   allow_failures:
     - os: osx
+    - env: TESTS=sawHMAC SAW_HMAC_TEST=sha384 SAW=true GCC6_REQUIRED=true
```

Comment:
That's a good question.  It looks like a time out - I can ask Galois why that might be.